pkg/storkctl/clusterpair.go

```diff
@@ -249,14 +252,22 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 				util.CheckErr(err)
 				return
 			}
-
-			srcClusterPair, err := generateClusterPair(clusterPairName, cmdFactory.GetNamespace(), dIP, dPort, destToken, dFile, projectMappingsStr, false)
+			fmt.Printf("Using PX-Service Endpoint of DR cluster to create clusterpair...\n")
+			ip, port, token, err := getClusterPairParams(dFile, dIP)
 			if err != nil {
+				err := fmt.Errorf("unable to create clusterpair from source to DR cluster. Err: %v", err)
 				util.CheckErr(err)
 				return
 			}
+			dIP = ip
+			if dPort == "" {
+				dPort = port
+			}
+			if destToken == "" {
```

Comment:
Since these are now optional to override the fetched values, we could mark them as (optional) or similar in the descriptions/help

Reply:
good idea, updated in cmd help